html/static/script.js

```diff
@@ -67,13 +67,52 @@ function saveNotes(element, uid) {
   })
 }
 
+function clearNotesMissing(element, dataset_name, campaign) {
+  $(element).parent().find('textarea').get(0).value = '';
+  let data = {"dataset_name": dataset_name, "campaign": campaign, "missing_nts": notes};
+  $.ajax({
```

Comment:
You probably don't need to do ajax call here. I think after you set
```
$(element).parent().find('textarea').get(0).value = '';
```
You can call
```
saveNotesMissing(element, dataset_name, campaign);
```
And it would take the empty string and save it in the database.